FileSyncandWOPI/Source/SharedTestSuite/SharedTestCase/S17_FileOperation.cs

```diff
@@ -226,6 +226,47 @@ public void TestCase_S17_TC03_ResourceIDNotChanged()
                     resourceID, resourceID2, "ResourceID MUST NOT change over the lifetime of a file, even if the URL of the file changes.");
             }
         }
+
+        /// <summary>
+        /// A method used to verify that an error when neither of the ResourceID and Url attributes identify valid files.
+        /// </summary>
+        [TestCategory("SHAREDTESTCASE"), TestMethod()]
+        public void TestCase_S17_TC04_ResourceIdDoesNotExist()
+        {
+            string invalidUrl = this.DefaultFileUrl + "Invalid";
+
+            // Initialize the service
+            this.InitializeContext(this.DefaultFileUrl, this.UserName01, this.Password01, this.Domain);
+
+            // Invoke "GetVersions"sub-request with correct input parameters.

```

Comment:
Please update comment to "Invoke "GetVersions"sub-request with invalid input parameters."